src/System.Private.Xml/src/System/Xml/Serialization/XmlReflectionImporter.cs

```diff
@@ -946,7 +946,7 @@ internal string XsdTypeName(Type type, XmlAttributes a, string name)
             if (a.XmlType != null && a.XmlType.TypeName.Length > 0)
                 typeName = a.XmlType.TypeName;
 
-            if (type.IsGenericType && typeName.Contains('{'))
+            if (type.IsGenericType && typeName.IndexOf('{') >= 0) // string.Contains(char) is .NetCore2.1+ specific (class imports Linq)
```

Comment:
See https://github.com/dotnet/corefx/pull/32249#issuecomment-421489707

Reply:
I am not sure if this is a concern. It doesn't look like we are compiling this for netstandard or older runtimes. What happens if we remove the `using System.Linq`? It seems to be unnecessary.

Reply:
OK, I will remove the `using System.Linq` and see if it's happy